src/Temporalio/Client/Schedules/ScheduleCalendarSpec.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.Collections.Generic;
+
+namespace Temporalio.Client.Schedules
+{
+    /// <summary>
+    /// Specification relative to calendar time when to run an action.
+    /// </summary>
+    /// <remarks>
+    /// A timestamp matches if at least one range of each field matches except for year. If year is
+    /// missing, that means all years match. For all fields besides year, at least one range must be
+    /// present to match anything.
+    /// </remarks>
+    public record ScheduleCalendarSpec
+    {
+        /// <summary>
+        /// Default range set for zero.
+        /// </summary>
+        public static readonly IReadOnlyCollection<ScheduleRange> Beginning = new[] { ScheduleRange.Zero };
```

Comment:
why is this public?

Reply:
Because others may want to use it, e.g. to reset something back to beginning that they may have changed.